darts/models/forecasting/regression_model.py

```diff
@@ -1124,36 +1123,37 @@ def _optimized_historical_forecasts(
             allow_autoregression=False,
         )
 
+        if predict_kwargs is None:
+            predict_kwargs = dict()
+
         # TODO: move the loop here instead of duplicated code in each sub-routine?
         if last_points_only:
             return _optimized_historical_forecasts_last_points_only(
                 model=self,
                 series=series,
                 past_covariates=past_covariates,
                 future_covariates=future_covariates,
-                num_samples=num_samples,
                 start=start,
                 start_format=start_format,
                 forecast_horizon=forecast_horizon,
                 stride=stride,
                 overlap_end=overlap_end,
                 show_warnings=show_warnings,
-                predict_likelihood_parameters=predict_likelihood_parameters,
+                **predict_kwargs,
```

Comment:
It looks like we don't pass the kwargs to regression model's predict() inside the optimized subroutines